src/SignIn/SignIn.tsx

```diff
@@ -0,0 +1,73 @@
+import React, {useState} from 'react';
+import {
+  Text,
+  StyleSheet,
+  useWindowDimensions,
+  Image,
+  KeyboardAvoidingView,
+  Platform,
+  Keyboard,
+} from 'react-native';
+
+import Logo from '../assets/images/logoTaqtile.png';
+import CustomInput from '../components/CustomInput';
+import CustomButton from '../components/CustomButton';
+
+const SignIn = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  const onSignInPressed = () => {
```

Comment:
Pra vc já se acostumar com as nossas convenções:

```suggestion
  const handleSignInPressed = () => {
```

As props de callback geralmente já são chamadas de `onAlgumaCoisa`, então as funções que implementam esses callbacks, a gente começa com `handle`